ci/ray_ci/test_upgrade_version.py

```diff
@@ -0,0 +1,103 @@
+from unittest import mock
+import sys
+import pytest
+import tempfile
+import os
+
+from ci.ray_ci.upgrade_version import (
+    list_java_files,
+    get_current_version,
+    upgrade_file_version,
+)
+
+
+def test_list_java_files():
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        os.mkdir(os.path.join(tmp_dir, "subdir_0"))
+        os.mkdir(os.path.join(tmp_dir, "subdir_1"))
+        os.mkdir(os.path.join(tmp_dir, "subdir_0/subdir_0_0"))
+        select_file_paths = [
+            "pom_template.xml",
+            "subdir_0/pom.xml",
+            "subdir_0/pom_template.xml",
+            "subdir_1/pom.xml",
+            "subdir_1/pom_template.xml",
+            "subdir_0/subdir_0_0/pom.xml",
+        ]
+        non_select_file_paths = [
+            "not_pom.xml",
+            "subdir_1/not_pom_template.xml",
+            "subdir_0/subdir_0_0/not_pom_template.xml",
+            "subdir_0/subdir_0_0/not_pom.xml",
+        ]
+        for file_path in select_file_paths + non_select_file_paths:
+            with open(os.path.join(tmp_dir, file_path), "w") as f:
+                f.write("")
+
+        assert list_java_files(tmp_dir) == {
+            os.path.join(tmp_dir, file_path) for file_path in select_file_paths
+        }
+
+
+@mock.patch("ci.ray_ci.upgrade_version.get_check_output")
+def test_get_current_version(mock_check_output):
+    mock_check_output.side_effect = [
+        "3.0.0.dev0 commit-sha",
+        "1.1.1.non_default commit-sha",
```

Comment:
some small things, could you:

- change the commit-sha to a string that looks like a commit digest? something like `0123456789abcdef` maybe?
- and change the `1.1.1.non_default` to maybe `2.2.0`, making it a more common use case that we can encounter?

Reply:
done